benchmarks/src/main/scala/scalaz/tests/TestBenchmark.scala

```diff
@@ -0,0 +1,20 @@
+package scalaz
+package tests
+
+import java.util.concurrent.TimeUnit
+import org.openjdk.jmh.annotations._
+import scala.Array
+import scala.Predef.{ wrapString, String }
+import scalaz.Predef._
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput))
+@OutputTimeUnit(TimeUnit.SECONDS)
+@Fork(value = 5, jvmArgsAppend = Array("-Xms2g", "-Xmx2g"))
+@Warmup(iterations = 0, batchSize = 1)
+@Measurement(iterations = 1, batchSize = 1, time = 1, timeUnit = TimeUnit.MILLISECONDS)
+class TestBenchmark {
+  @Benchmark
+  def allTests(): Unit =
+    TestMain.main(Array.empty)
```

Comment:
I'd be curious about benchmarking individual tests but nm

Reply:
You can definitely do that; just add a method like `allTests` that calls into a particular test suite.